app/assets/javascripts/short-form/ShortFormNavigationService.js.coffee

```diff
@@ -48,7 +48,7 @@ ShortFormNavigationService = (
   # We show it for all rentals, and for Sale listings that are
   # accessible units only.
   Service.showHouseholdPrioritiesPage = (listing) ->
-    _.includes(ListingConstantsService.HOUSEHOLD_PRIORITIES_LISTINGS_IDS, listing.Id) ||
+    (listing && _.includes(ListingConstantsService.HOUSEHOLD_PRIORITIES_LISTINGS_IDS, listing.Id)) ||
```

Comment:
@jimlin-sfgov want to make sure you get eyes on this change - shouldn't change any functionality because it's just checking for existence